haiku-apps/projectconceptor/projectconceptor-1.0.0.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A Haiku diagramming application like Microsoft Visio"
+DESCRIPTION="ProjectConceptor is a modular application for Haiku and BeOS \
+designed to view, edit and process information which can be represented by a \
+graph structure."
+APP="ProjectConceptor"
+HOMEPAGE="https://github.com/Paradoxianer/$APP"
+COPYRIGHT="2017 Paradoxianer"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="131fc9f8ae91d72d2d463604b68215f1b39a77e3"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="d0e62996bf7d17b462a7f25b3c9f80b252a8f395aefb34e078575a16536f7472"
+SOURCE_FILENAME="$APP-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="$APP-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 !x86_64"
+
+PROVIDES="
+	projectconceptor = $portVersion
+	app:$APP = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:find
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	make -C src OBJ_DIR=objects $jobArgs
+}
+
+INSTALL()
+{
+	install -d $appsDir/$APP/Plugins
+	install src/app/$APP $appsDir/$APP
+	addAppDeskbarSymlink $appsDir/$APP/$APP
+
+	cd src/plugins
+	install $(find * -mindepth 2 -type f -executable) $appsDir/$APP/Plugins
```

Comment:
Is there any reason to use `*` here?

```diff
-	install $(find * -mindepth 2 -type f -executable) $appsDir/$APP/Plugins
+	install -t $appsDir/$APP/Plugins $(find . -mindepth 2 -type f -executable)

Reply:
If you use the -t format, the line will exceed 80 characters? I prefer the shorter format if they are equivalent. As to why using ‘*’ not ‘.’, compare the outputs of “find” and you will see why.

Reply:
I think we can keep what we have right now:
```
	install $(find * -mindepth 2 -type f -executable) $appsDir/$APP/Plugins
```
Or maybe use a mix:
```
	install -t $appsDir/$APP/Plugins $(find * -mindepth 2 -type f -executable)
```